docs/api/supported-platforms.md

```diff
@@ -87,43 +67,80 @@ pnpm install @finos/fdc3
+
+The FDC3 NPM module implements the `getAgent()` function defined by WCP and can return an injected Desktop Agent, a Desktop Agent Proxy, or other Desktop Agent implementation enabled by a non-standard interface.
+
+Hence, FDC3 apps SHOULD obtain access to a `DesktopAgent` object (`fdc3`) by importing or loading the `@finos/fdc3` library and then calling the provided `getAgent()` function, ensuring that they can support any of the standardized interfaces.
 
-await fdc3.raiseIntent('ViewAnalysis', {
+:::info
+
+In prior versions of FDC3 (<= 2.1) Apps were required to use the 'Desktop Agent Preload' interface, i.e. they relied on the existence of the `window.fdc3` object, which meant that apps running in a standard web browser had to import libraries specific to the Desktop Agent implementation in use. From FDC3 2.2 onwards the 'Desktop Agent Proxy' interface is available, which allows apps in a standard web browser to connect to any Desktop Agent that implements that interface.
+
+Hence, from FDC3 2.2 onwards apps SHOULD call the `getAgent()` to retrieve a `DesktopAgent` API interface.
+
+:::
+
+As web applications can navigate to or be navigated by users to different URLs and become different applications, validation of apps identity is often necessary. The web application's current URL is passed to web browser-based Desktop Agents to allow them to establish the app's identity - usually connecting it with an App Directory record already known to the Desktop Agent. For more details on identity validation see the Identity Validation section of the [Web Connection Protocol (WCP)](specs/webConnectionProtocol).
+
+### Usage
+
+Once you've retrieved a `DesktopAgent` interface, there are two main ways FDC3 can be used from web applications:
+
+#### 1. Direct Usage
+
+Simply use the interface you've retrieved and address the API directly:
+
+```js
+async function sendData(fdc3: DesktopAgent) {
+  await fdc3.broadcast({
     type: 'fdc3.instrument',
     id: { ticker: 'AAPL' }
-});
+  })
+}
+
+const fdc3: DesktopAgent = await getAgent();
+await sendData(fdc3);
 ```
 
-It also includes a helper function you can use to wait for FDC3 to become available:
+#### 2. es6-style Function Wrappers
 
-```javascript
-import { fdc3Ready, addIntentListener } from '@finos/fdc3'
+The npm package provides a wrapper around FDC3, allowing you to use it with ES6 import syntax:
 
-await fdc3Ready();
+```javascript
+import {raiseIntent} from '@finos/fdc3';
 
-const listener = await addIntentListener('ViewAnalysis', instrument => {
-  // handle intent
+await raiseIntent('ViewAnalysis', {
```

Comment:
How does this work?  Does it use window.fdc3?  If so, we might need to change this as that might not always be set with getAgent.

Reply:
Yeah it currently uses window.fdc3 - so with the flag for setting window.fdc3, it'll continue to just work. However, we should update this example to show the get agent call. Added a suggestion above. Still not setting parameters though as the defaults should work for most apps/cases... that includes setting `window.fdc3`

Oh and a call to fdc3Ready() is not needed as getAgent is handling that - I'll add a note to the docs for `fdc3Ready`.